pkg/lib/machine-change-auth.html

```diff
@@ -71,6 +71,21 @@ <h4 class="modal-title" translate="yes">Log in to {{host}}</h4>
                         {{#public-key}}<div class="keys"><div class='dialog-wait-ct pull-left'><div class='spinner spinner-sm'></div><span translate="yes">Checking for public keys</span></div></div>{{/public-key}}
                 </td>
             </tr>
+            {{#show_locked_identity}}
+            <tr class="login-locked">
+                <td translate="yes" class="top">Locked</td>
+                <td class="top">
+                    <div class="key-locked">
+                        <div class="locked-identity"></div>
+                        <div translate="yes">Enter passphrase to add identity to ssh-agent</div>
```

Comment:
So that means we don't even redirect the user to the existing "Authentication" dialog, but it's done inline in the Troubleshooting dialog? That's a nice workflow indeed! :+1 I'm sure a lot of users don't even know that the Auth dialog exists.

Perhaps this actually could just re-use the existing dialog somehow, but as the code is written now, let's use it.

Reply:
Yea it uses the same `Keys` function from `pkg/lib/credentials.js`, but not the dialog itself.